scripts/ai/AIDriveStrategyUnloadCombine.lua

```diff
@@ -723,6 +723,36 @@ function AIDriveStrategyUnloadCombine:updateFillUnitAutoAimTarget(superFunc, fil
     end
 end
 
+------------------------------------------------------------------------------------------------------------------------
+-- Who I am?
```

Comment:
Maybe adding a small description might be got. 
``Which vehicle type am I unloading?`` or something similar.

Reply:
added